CONTRIBUTING.md

```diff
@@ -131,8 +131,10 @@ Some common make targets are:
   & everything builds in the correct manner.
 - `make check-stage1-std NO_REBUILD=1` - test the standard library without
   rebuilding the entire compiler
-- `make check TESTNAME=<path-to-test-file>.rs` - Run a single test file
-- `make check-stage1-rpass TESTNAME=<path-to-test-file>.rs` - Run a single
+- `make check TESTNAME=<name-of-test-to-run>` - Run a single test file
+  - `TESTNAME` should be the fully qualified name of the test function, e.g.
```

Comment:
Actually I think that `TESTNAME` is basically just a string match against the name of each test. For example all of the run-pass tests are named after the basename of their files, and the libstd tests are named based on modules and such. So for example the test you mentioned below would also be matched with the filters `collections`, `capacity`, `hash::map`, etc.